### PR TITLE
Feature/data base provision

### PR DESCRIPTION
**Pull request by Summer0410**
Connected the app back end to mongodb database

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python3"
+}
```
Comment (BenxinNiu) [inline .vscode/settings.json]: please update .gitignore file to exclude .vscode

File: src/app.py [hunk 2]
```diff
@@ -1,5 +1,15 @@
+from flask import Flask
 import connexion
+from flask_pymongo import pymongo
+import repositories.db as DB
```
Comment (BenxinNiu) [inline src/app.py:4]: `flask_pymongo` is not found in `requirements.txt` which means you might have installed globally on your local dev. Pls fix this

File: src/app.py [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
 
 app = connexion.App(__name__, specification_dir='./')
-app.add_api('swagger.yaml')
-app.run(port=8822)
```
Comment (BenxinNiu) [inline src/app.py:5]: Also, why is the swagger file removed?

File: src/repositories/db.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+import connexion
+from flask_pymongo import pymongo
+from .connection_string import MONGO_CONNECTION_STRING
+from app import app
+
+client = pymongo.MongoClient(MONGO_CONNECTION_STRING)
+db = client.get_database('db_sensor')
+collection_sensor = db.collection_sensor
```
Comment (BenxinNiu) [inline src/repositories/db.py:9]: nice work on creating "singleton" for db connection

**Review by BenxinNiu — CHANGES_REQUESTED** [review]

File: src/app.py (hunk 3 above)
Comment (Summer0410) [reply src/app.py:5]: Made changes regarding to all your comments, please have a look

**Review by BenxinNiu — APPROVED** [review]
Missing configuration for SSL certificate... but will fix later

**Pull request merged**